theme/theme.scss

```diff
@@ -0,0 +1,17 @@
+figure {
+	&.ck-widget.image {
+		text-align: center;
+
+		img {
+			pointer-events: none;
+		}
+
+		&.ck-widget_selected img, &.ck-widget_selected:hover img{
```

Comment:
Besides, image selected in a blurred editor should not have a blue outline, but grey.